modules/EditWindow/PfamWindow.pm

```diff
@@ -96,7 +96,11 @@ sub result_url {
 
 sub widg {
     my ($self, $name) = @_;
-    return $self->{_widg}->{$name} or die "No widg($name)";
+
+    if (! $self->{_widg}->{$name}) {
```

Comment:
I would change this to a definedness check, on the off chance someone decides to name the widget "0" (which is defined yet evaluates to false).

Reply:
I can change that